extensions/interactions/guppy.html

```diff
@@ -41,6 +39,17 @@
     color: #000000 !important;
   }
 
+  .base {
```

Comment:
You're right, I had a thorough look and it seems that there are other classes that use the base class which seem to be affected as well. However, the issue seems to come exactly because of the width and white space properties given by the base class. I'll try and see if I can find an alternate way to solve this.

Reply:
Thanks, sounds good. Just a note: even if there weren't other classes that use the base class, my comment would still apply, because it's too easy for someone to use that class in the future as well. (So for example it applies to .mord below as well.) That's why we try to use specificity when writing CSS selectors, to constrain the style updates to a particular component.